<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgSBaseReg</class>
 <widget class="CItemDlg" name="CDlgSBaseReg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Base Stress Region Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Shaps:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboMethod"/>
     </item>
     <item>
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>CheckRegion:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnInCircle">
       <property name="text">
        <string>InnerCircle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOutCircle">
       <property name="text">
        <string>OutterCircle</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Edge</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinEdge">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="minimum">
        <number>-255</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Gray</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinGray">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Area</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinArea">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>SideDistance</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinSideDistance">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Anisometry</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinRab">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CItemDlg</class>
   <extends>QWidget</extends>
   <header>citemdlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>accept()</slot>
  <slot>reject()</slot>
 </slots>
</ui>
